homeassistant/components/qbittorrent/sensor.py

```diff
@@ -83,6 +121,109 @@ def format_speed(speed):
         native_unit_of_measurement=UnitOfDataRate.KIBIBYTES_PER_SECOND,
         state_class=SensorStateClass.MEASUREMENT,
         value_fn=lambda data: format_speed(data["server_state"]["up_info_speed"]),
+        attrs_fn=None,
+    ),
+)
+
+
+TORRENT_SENSOR_TYPES: tuple[QBittorrentSensorEntityDescription, ...] = (
+    QBittorrentSensorEntityDescription(
+        key=SENSOR_TYPE_TORRENTS,
+        name="Torrents",
+        icon="mdi:cog-transfer",
+        state_class=SensorStateClass.MEASUREMENT,
+        value_fn=lambda data: count_torrents(data, None),
+        attrs_fn=lambda data: list_torrents(data, None),
+    ),
+    QBittorrentSensorEntityDescription(
+        key=SENSOR_TYPE_TORRENTS_DOWNLOADING,
+        name="Torrents Downloading",
+        icon="mdi:cloud-arrow-down",
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
```suggestion
        state_class=SensorStateClass.MEASUREMENT,
        entity_registry_enabled_default=False,
```

Reply:
@joostlek I've done this and set it conditionally to the value of the preferences, however now each entity stays disabled once the config option is switched, even after restarting homeassistant. (Previously, restarting homeassistant would cause the new entities to be created after toggling the option for adding torrent status metadata.) 

Do you think this extra work of conditionally adding the entities is worth it, or should they just always be enabled by the integration? (It doesn't require an additional api call or anything from what was existing)